1-basic/dubbo-samples-api/src/main/java/org/apache/dubbo/samples/provider/Application.java

```diff
@@ -28,7 +28,13 @@ public class Application {
     private static final String ZOOKEEPER_PORT = System.getProperty("zookeeper.port", "2181");
     private static final String ZOOKEEPER_ADDRESS = "zookeeper://" + ZOOKEEPER_HOST + ":" + ZOOKEEPER_PORT;
 
+    private static final String QOS_PORT_PARAM_KEY = "dubbo.application.qos-port";
+    private static final String QOS_PROVIDER_PORT = "33333";
+
+
     public static void main(String[] args) {
+        System.setProperty(QOS_PORT_PARAM_KEY, QOS_PROVIDER_PORT);
```

Comment:
Set in `ApplicationConfig`

Reply:
I have modified this code. Is it the correct way to use Dubbo's ApplicationConfig to accomplish this?